internal/log/log.go

```diff
@@ -84,3 +89,29 @@ func Fatal(vals ...interface{}) {
 func Fatalf(format string, vals ...interface{}) {
 	logger.Fatalf(format, vals...)
 }
+
+var (
+	retryOut = func(fn func(vals ...interface{}) error, vals ...interface{}) {
```

Comment:
why are you define these functions?
maybe, it should be define in retry package

Reply:
Yes, but there was a problem with the import cycle ...
Please wait for a while as we will think about measures

Reply:
I fixed